packages/api/src/command/medical/patient/__tests__/consolidated-get.test.ts

```diff
@@ -53,3 +56,30 @@ describe("getConsolidatedPatientData", () => {
     expect(fhir_searchResourcePages).toHaveBeenCalledTimes(1);
   });
 });
+
+
+function loadPatientData(data: any): PatientData {
+  return {
+    ...data,
+    genderAtBirth: data.genderAtBirth as GenderAtBirth,
+  };
+}
```

Comment:
Let's avoid using `any` and `as` when possible, please. Happy to chat and explain why. :)

In this case, just changing how we declare the test data allows us to use it typed here and avoid runtime errors - sure, this is only a test, but its a good practice and minimizes back-and-forth with commits and CI/CD errors.

From:

```ts
export const testPatientData = {
```

...to:

```ts
export const testPatientData: {
  sampleSearch: PatientData[];
  sampleInclusions: PatientData[];
  sampleExclusions: PatientData[];
} = {
```